pkg/sdk/go/sdk.go

```diff
@@ -292,7 +292,7 @@ type SDK interface {
 	//	pm := sdk.PageMetadata{
 	//		Offset: 0,
 	//		Limit:  10,
-	//		Permission: "edit", // available Options:  "administrator", "delete", edit", "view", "share", "owner", "admin", "editor", "viewer"
+	//		Permission: "edit", // available Options:  "delete", edit", "view", "share", "admin"
 	//	}
```

Comment:
We also support to list using relations, Please change like below and test the listing API 
```suggestion
	//		Permission: "edit", // available Options:  "administrator", "delete", edit", "view", "share", "owner", "admin", "editor", "contributor"
```

Reply:
Please revert back to original and change only `viewer` to `contributor` everywhere in SDK 
We can resolve them in seperate issue https://github.com/absmach/magistrala/issues/2275